pkg/vif/routing/routing_linux.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/telepresenceio/telepresence/v2/pkg/iputil"
 )
 
-const findInterfaceRegex = `^(local\s)?[0-9.]+(\s+via\s+(?P<gw>[0-9.]+))?\s+dev\s+(?P<dev>[a-z0-9-]+)\s+src\s+(?P<src>[0-9.]+)`
+const findInterfaceRegex = `^(local\s)?[0-9.]+(\s+via\s+(?P<gw>[0-9.]+))?\s+dev\s+(?P<dev>[a-z0-9-]+)\s+(table [0-9]+\s+)?src\s+(?P<src>[0-9.]+)`
```

Comment:
My workstation have routes with `table local`, so `[0-9]` is too narrow. Example:
```
local 192.168.50.10 dev lo table local src 192.168.50.10 uid 0
```